### PR TITLE
Api-Buildout: Updating data pulled from Prismic for About Page

### PR DESCRIPTION
**Pull request by jzhzhang66**
Pulled in the following data for about page: 
- image for each value 
- main resources titles / description / links
- more resources links 
- mission title / description
- top & bottom quote 
- the alleyway names title / description / text 

Created new component MainResource to render each main resource 

Added getMainResources in api.js to get main resources given list of ids

### CODE REVIEW

**Review by josephmannis — APPROVED** [review]
WAIT I commented on the wrong pull request, sorry – real review coming in one sec.

File: components/MainResource.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { H3 } from "../style/typography";
+export default function MainResource({ mainResourceData }) {
+
+  const {
+    resourceTitle,
+    resourceLink,
+  } = mainResourceData;
+
+  return (
+    <>
```
Comment (josephmannis) [inline components/MainResource.js]: I don't think you need a fragment here

File: pages/about.js [hunk 2]
```diff
@@ -1,40 +1,74 @@
...
+        theAlleywayNamesMeaning.map((m) => <P key={m.id}>{getString(m.meaning)}</P>)
+      }
+      {
+        mainResourcesData.map((r) => (
+          <MainResource key={r.id} mainResourceData={r} />
+        ))
+      }
+      <br />
+      {
+        moreResources.map((r) => <P>{r.resource.url}</P>)
```
Comment (josephmannis) [inline pages/about.js]: 🔑

**Review by josephmannis — APPROVED** [review]
Very nice!

File: pages/about.js [hunk 3]
```diff
@@ -1,40 +1,74 @@
...
       <H3>{getString(valuesDescription)}</H3>
-{
+      {
         valuesData.map((v) => (
-            <Value key={v.id} valueData={v.data}/>
+          <Value key={v.id} valueData={v.data} />
         ))
       }
+      <H3>{getString(theAlleywayNamesTitle)}</H3>
+      {
```
Comment (jean-zhang) [inline pages/about.js]: formatting?

File: pages/about.js [hunk 4]
```diff
@@ -45,13 +79,18 @@ export async function getStaticProps() {
   const footerData = await getFooter();
   const navigationData = await getNavigation();
   const valuesData = await getValues();
+  const mainResourceIds = aboutPageData.main_resources.map((item) => {
```
Comment (jean-zhang) [inline pages/about.js]: nit: `map((item) => (item.resource.id));`

**Review by jean-zhang — APPROVED** [review]

**Pull request merged**